frontend/src/components/staff/CreatePackage.jsx

```diff
@@ -76,6 +76,12 @@ export default function CreatePackage({ refresh, onRefresh }) {
         }));
         setInventoryData(dataWithKey);
         setLoading(false);
```

Comment:
this setLoading can move to after the fetch

Reply:
<img width="452" alt="Screenshot 2023-07-04 at 8 44 13 PM" src="https://github.com/It-s-Saturday/Fornear/assets/57148990/e0776c7c-760d-4a35-bb23-5e17ebac1be9">

So, after fetching loading should be false?